File: Fibbing.py
a = 0
b = int(input("enter the starting value: "))
amount = int(input("How many places should it go up to: "))
z = 0

total = 0
list = []

for i in range(amount):
    z = a + b
    total = total + z
    list.append(z)
    a = b
    b = z

print(list)
print(f"The total amount of all the numbers is: {total}")
print(f"Your array backwards is {list[::-1]}")

File: fibbing.c
#include <stdio.h>

int main(){
    int a = 0;
    int b;
    int z;
    int total;
    int amount;

    printf("What should be the starting number: ");
    scanf("%d",&b);
    printf("How far should we go up: ");
    scanf("%d",&amount);

    for (int i = 0; i < amount; i++)
    {
        z = a + b;
        printf("%d ",z);
        a = b;
        b = z;
    }
    
}